scripts/rddepman.ts

```diff
@@ -0,0 +1,211 @@
+// A cross-platform script to create PRs that bump versions of dependencies.
+
+import { spawnSync } from 'child_process';
+import path from 'path';
+
+import { Octokit } from 'octokit';
+import { LimaAndQemu, AlpineLimaISO } from 'scripts/dependencies/lima';
+import { MobyOpenAPISpec } from 'scripts/dependencies/moby-openapi';
+import * as tools from 'scripts/dependencies/tools';
+import { WSLDistro, HostResolverHost } from 'scripts/dependencies/wsl';
+import {
+  DependencyVersions, readDependencyVersions, writeDependencyVersions, Dependency, AlpineLimaISOVersion, getOctokit,
+} from 'scripts/lib/dependencies';
+
+const MAIN_BRANCH = 'main';
+const GITHUB_OWNER = process.env.GITHUB_REPOSITORY_OWNER || 'rancher-sandbox';
+const GITHUB_REPO = process.env.GITHUB_REPOSITORY || 'rancher-desktop';
```

Comment:
`GITHUB_REPOSITORY` includes the owner, per [documentation](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables).

Reply:
Nice catch. Strangely enough, there is no `GITHUB_REPOSITORY_NAME` env var - I'll have to add one in the github workflow.

Reply:
Actually, it doesn't seem that there is a reliable way of getting the repo name, other than parsing it out of `GITHUB_REPOSITORY`. So that's what I'll do.